### PR TITLE
fix(tag): revert to using refname since creator date is inconsistent

### PR DESCRIPTION
**Pull request by humblerookie**
revert to using refname for fetching last tag since creator date was causing issues

### CODE REVIEW

**Review by notbenoit — APPROVED** [review]

File: lib/fastlane/plugin/fueled/helper/fueled_helper.rb [hunk 1]
```diff
@@ -16,7 +16,7 @@ def self.nil_or_empty(input)
       # Returns the last tag in the repo, one can filter tags by specifying the parameter
       def self.fetch_last_tag(filter:)
         tag = nil
-        git_cmd = "git tag -l --sort=-creatordate"
+        git_cmd = "git tag -l --sort=-refname"
```
Comment (pulkit-vaid) [inline lib/fastlane/plugin/fueled/helper/fueled_helper.rb:19]: `refname` or `version:refname`. 
not sure if the result is gonna be any different 🤔

Comment (pulkit-vaid) [reply lib/fastlane/plugin/fueled/helper/fueled_helper.rb:19]: `refname`:
```
0.2.11
0.2.101
0.2.1
0.2.0
```

`v:refname`
```
0.2.101
0.2.11
0.2.1
0.2.0
```

I guess we need `v:refname` or `version:refname`, `refname` does a lexical sort, which could end up returning incorrect tags.

Comment (pulkit-vaid) [inline lib/fastlane/plugin/fueled/helper/fueled_helper.rb:19]:
```suggestion
        git_cmd = "git tag -l --sort=-v:refname"
```

Comment (pulkit-vaid) [inline lib/fastlane/plugin/fueled/helper/fueled_helper.rb:19]:
```suggestion
        git_cmd = "git tag -l --sort=-v:refname"
```

**Pull request merged**